deploy/docker-compose-weave/docker-compose.yml

```diff
@@ -13,6 +13,7 @@ services:
       - external
       - secure
       - internal
+      - backoffice
```

Comment:
The front-end exposes a `/health` endpoint which pings all the other services health endpoints. `shipping`, `queue-master`, and `payment` endpoints aren't exposed through the front-end. Rather than adding and exposing those endpoints we have front-end ping and aggregate all the responses into one.